sbc_compassion/models/contracts.py

```diff
@@ -71,6 +76,20 @@ def _compute_last_letter(self):
             except TypeError:
                 contract.last_letter = -1
 
+    def _compute_write_for_birthday_alert(self):
+        for contract in self:
+            if contract.last_letter == -1 or contract.last_letter > 30:
+                next_birthday = fields.Date.from_string(
+                    contract.child_id.birthdate).replace(year=date.today.year)
+
+                # take next year birthday
+                # if birthday already pass for this year
+                if next_birthday - date.today() < timedelta(days=0):
+                    next_birthday.replace(year=next_birthday.year+1)
+
+                write_for_birthday_alert = \
+                    next_birthday - date.today() > timedelta(days=30)
```

Comment:
```suggestion
                    next_birthday - date.today() <= timedelta(days=90)
```
I think the test should be the opposite, we want to check that the birthday is in less than 3 months.